### ISSUE TITLE
Is there any plan to support the model deployment of 3D target detection? 

### ISSUE THREAD

**Issue by jianghaijun007** [open-mmlab/mmdeploy#334]
Is there any plan to support the model deployment of 3D target detection? When deploying monoflex, we encountered many pytorch operations that tensorrt does not support, such as edge_ indices. view(b, -1, 1, 2). float(),output_ cls[k, :, edge_indice_k[:, 1], edge_ indice_ k[:, 0]] += edge_ cls_ Output [K,:,: edge_lens [k]] and so on。

**Comment by lvhan028** [comment]
We've supported CenterPoint and PointPillars from MMDetection3D. @ZwwWayne do we have any plan to support monoflex?

**Comment by jianghaijun007** [comment]
@ZwwWayne do you have any plan to support monoflex?

**Comment by ZwwWayne** [comment]
Yes, we do have plans to support the model deployment of some mono3D detectors. However, we do not have developers to do that in the next two months. Therefore, progress will be made starting from July.
